.gitignore

```diff
@@ -17,11 +17,14 @@ share/python-wheels/
 nipoppy/notebooks/.ipynb_checkpoints/*
 
 # dataset-specific files
-bids_filter.json 
+bids_filter.json
 workflow/proc_pipe/fmriprep/scripts/fmriprep_sge.sh
 workflow/proc_pipe/fmriprep/scripts/fmriprep_slurm.sh
 
 # logs/testing
 *.out
 *.log
 env/
+
```

Comment:
```suggestion

# testing
sourcedata/

```

Reply:
running the tests dumped this folder in my repo

Reply:
Hm that's unexpected, and doesn't happen when I run the rests locally. Are you sure this is not from your BIDSCOIN testing? I noticed that you also [updated the `.gitignore`](https://github.com/neurodatascience/nipoppy/blob/3b6caeed18f801ccab88099c6cd750c33bcd31ce/.gitignore#L28) in the BIDSCOIN PR #204

Reply:
ha you may be right

please ingore